src/eval/ops.rs

```diff
@@ -423,3 +426,91 @@ pub fn contains(lhs: &Value, rhs: &Value) -> Option<bool> {
         _ => Option::None,
     }
 }
+
+pub fn pipe(b: &Binary, vm: &mut Vm) -> SourceResult<Value> {
+    use crate::diag::At;
+    let rhs = b.rhs();
+    let span = b.span();
+    let mut piped_args: Vec<Value>;
+    if let Some(lhs) = b.as_untyped().cast_first_match::<ast::PipedSpread>() {
+        let Some(expr) = &lhs.as_untyped().cast_first_match::<Expr>() else {unreachable!()};
+        let expr = expr.eval(vm)?;
+        match expr {
+            Array(arr) => {
+                piped_args = arr.into_iter().collect();
+            }
+            // Dict(dict) => {
```

Comment:
are statements like
`(x : 4, y : "string value").. |> f()`
really that clear or useful? I don't think so and there are easy workaround with pipes if you still wanna do it.